src/modules/Docker/DockerModule.tsx

```diff
@@ -17,6 +18,7 @@ export default function DockerMenuButton(props: any) {
   const [selection, setSelection] = useState<Docker.ContainerInfo[]>([]);
   const { config } = useConfigContext();
   const { classes } = useCardStyles(true);
+  useHotkeys([['mod+B', () => setOpened(!opened)]]);
```

Comment:
Is that documented?

Reply:
Done: https://homarr.dev/docs/advanced/keyboard-shortcuts